src/components/Foremost/Foremost.tsx

```diff
@@ -149,8 +153,8 @@ const ForemostTool = () => {
                 {/* Advanced Mode Switch */}
                 <Switch
                     label="Advanced Mode"
-                    checked={checkedAdvanced}
-                    onChange={(e) => setCheckedAdvanced(e.currentTarget.checked)}
+                    checked={form.values.checkedAdvanced}
```

Comment:
Hi @YashKanodia,

Please review the below exception for the return type.

To fix this you can:

 Check the interface and ensure that it exists.